plugins/gcov/README.md

```diff
@@ -35,12 +56,85 @@ for additional options and information. All generated reports will be found in `
   # Specify one or more reports to generate.
   # Defaults to HtmlBasic.
   :reports:
-    - HtmlBasic     # Make an HTML summary report. (gcovr --html)
-    - HtmlDetailed  # Make an HTML report with line by line coverage of each source file. (gcovr --html-details)
-    - Text          # Make a Text report, which may be output to the console or a file.
-    - Cobertura     # Make a Cobertura XML report. (gcovr --xml)
-    - SonarQube     # Make a SonarQube XML report. (gcovr --sonarqube)
-    - JSON          # Make a JSON report. (gcovr --json)
+    # Make an HTML summary report.
+    # Supported utilities: gcovr (--html), Report Generator
```

Comment:
`# Supported utilities: gcovr (--html), ReportGenerator (-reporttypes:HtmlSummary)` will be better? Or maybe just `# Supported utilities: gcovr, Report Generator` and leave usage flags. The same for the other options?

Reply:
I removed the `(--html)` and similar. I thought it looked funny too.